src/sentry/auth/superuser.py

```diff
@@ -108,7 +111,10 @@ def superuser_has_permission(
         return True
 
     # if we aren't enforcing superuser read-write, then superuser always has access
-    if not features.has("auth:enterprise-superuser-read-write", actor=request.user):
+    if not (
+        features.has("auth:enterprise-superuser-read-write", actor=request.user)
+        or options.get("superuser.read-write.ga-rollout")
+    ):
```

Comment:
this is the only other code change besides adding the option, everything else is modifying test decorators